inference-engine/tests/functional/plugin/shared/include/subgraph_tests/multiply_concat_reshape_reshape_concat.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
```

Comment:
I'm sorry but do you actually use something form  `ngraph_helpers.hpp` here in this header?